README.md

```diff
@@ -1 +1,27 @@
-# esp-rom-elfs
+# ESP32 series ROM ELF files
+
+The [Releases](https://github.com/espressif/esp-rom-elfs/releases) page of this repository is used to publish the ELF files of ROMs used by ESP32 series of chips.
+
+The releases will be updated whenever a new Espressif chip or a chip revision is released.
+
+## Choosing the ROM ELF file
+
+In the release archives, you can find files with two filename patterns:
+
+1. <CHIP_NAME>_rom.elf — for chips where the ROM code is the same in all chip revisions.
+2. <CHIP_NAME>_rev<REV>_rom.elf — for chips where the ROM code differs between chip revisions. You need to use the version of ROM with the revision equal to the revision of your chip. If such file is not available, use the next lower one.
```

Comment:
@Lapshin what do you think about _always_ adding `_rev0_` for the chips which only have one ROM revision? The benefit would be that there is no "special case" for handling such chips. Also if the new revision is added, we wouldn't have to modify the existing file name, will just add a new one.

Reply:
@igrr , yes, it would be more convenient for all. Let me do this.

Reply:
@Lapshin I have updated this part, PTAL again.

Reply:
LGTM! Thank you, @igrr